src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -4,24 +4,23 @@
 import mate.academy.spring.dao.UserDao;
 import mate.academy.spring.model.User;
 import mate.academy.spring.service.UserService;
-import mate.academy.spring.util.HashUtil;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
-    private static final int SALT_LENGTH = 10;
     private final UserDao userDao;
+    private final PasswordEncoder passwordEncoder;
 
-    public UserServiceImpl(UserDao userDao) {
+    public UserServiceImpl(UserDao userDao, PasswordEncoder passwordEncoder) {
         this.userDao = userDao;
+        this.passwordEncoder = passwordEncoder;
     }
 
     @Override
     public User add(User user) {
-        String salt = HashUtil.getSalt(SALT_LENGTH);
-        String securePassword = HashUtil.generateSecurePassword(user.getPassword(), salt);
-        user.setPassword(securePassword);
-        user.setSalt(salt);
+        String encode = passwordEncoder.encode(user.getPassword());
+        user.setPassword(encode);
```

Comment:
do not forget about the vars naming
```suggestion
        String securePassword = passwordEncoder.encode(user.getPassword());
        user.setPassword(securePassword);
```